src/tools/schema2kotlin.ts

```diff
@@ -35,6 +38,16 @@ const typeMap = {
   'B': {type: 'Boolean', decodeFn: 'decodeBool()', defaultVal: 'false', schemaType: 'FieldType.Boolean'},
 };
 
+function typeFor(name: string): string {
```

Comment:
Generally, we've been using the `typeMap` table for this kind of thing... though it's getting out of hand. 
e.g. `typeMap[input].type`

Reply:
This might be a good opportunity to find a cleaner means of lookup / type-association.

Reply:
I think I'll have to push this on off once more as there's a bigger task here for handling more complex types that is out of scope for this change.